pysph/base/tree/point_octree.mako

```diff
@@ -0,0 +1,367 @@
+//CL//
+<%def name="preamble(data_t)" cached="True">
+    #define IN_BOUNDS(X, MIN, MAX) ((X >= MIN) && (X < MAX))
+    #define NORM2(X, Y, Z) ((X)*(X) + (Y)*(Y) + (Z)*(Z))
+    #define MIN(X, Y) ((X) < (Y) ? (X) : (Y))
+    #define MAX(X, Y) ((X) > (Y) ? (X) : (Y))
+    #define AVG(X, Y) (((X) + (Y)) / 2)
+    #define ABS(X) ((X) > 0 ? (X) : -(X))
+    #define EPS 1e-6
+    #define INF 1e6
```

Comment:
I don't like this definition of INF!  Why this number? Same with EPS.  If there is a good reason for it, I think a comment is in order.